commit 14e2ace138a6a168f20f3d83ec36675189206734
Author: Alexander Kurtakov <akurtako@redhat.com>
Date:   Wed Aug 20 23:45:15 2014 +0300

    Bug 441791 - Remove motif specific things from codebase
    
    Cleanup examples.
    
    Change-Id: I3d1a204deb856691cb3e0a1a557ca9121eba69b0
    Signed-off-by: Alexander Kurtakov <akurtako@redhat.com>

0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
0	2	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
2	12	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/texteditor/text.txt
1	1	examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
index 8af3e9a..99b6f03 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
@@ -227,8 +227,6 @@ public class AlphaTab extends AnimatedGraphicsTab {
 	static Font createFont(Device device, int points, int style) {		
 		if(SWT.getPlatform() == "win32") {
 			return new Font(device, "Verdana", points, style);	
-		} else if (SWT.getPlatform() == "motif") {
-			return new Font(device, "Times", points, style);		
 		} else if (SWT.getPlatform() == "gtk") {
 			return new Font(device, "Baekmuk Batang", points, style);		
 		} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
index 4e57e23..16d099b 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
@@ -256,8 +256,6 @@ public class CountDownTab extends AnimatedGraphicsTab {
 	static String getPlatformFontFace(int index) {
 		if (SWT.getPlatform() == "win32") {
 			return new String[] { "Courier", "Impact" }[index];
-		} else if (SWT.getPlatform() == "motif") {
-			return new String[] { "Courier", "URW Gothic L" }[index];
 		} else if (SWT.getPlatform() == "gtk") {
 			return new String[] { "Courier", "Baekmuk Headline" }[index];
 		} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
index 2568a7d..50ec191 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
@@ -380,8 +380,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
index c7d81b6..4b6f06f 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
@@ -289,8 +289,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
index b0a883d..8855e21 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
@@ -138,8 +138,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFontFace(int index) {	
 	if(SWT.getPlatform() == "win32") {
 		return new String [] {"Arial", "Impact", "Times", "Verdana"} [index];	
-	} else if (SWT.getPlatform() == "motif") {
-		return new String [] {"URW Chancery L", "URW Gothic L", "Times", "qub"} [index];		
 	} else if (SWT.getPlatform() == "gtk") {
 		return new String [] {"URW Chancery L", "Baekmuk Batang", "Baekmuk Headline", "KacsTitleL"} [index];
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
index f1fe5d6..7036eb7 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
@@ -101,8 +101,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
index 493f806..e00d369 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
@@ -188,8 +188,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
index fcf6a98..23910bd 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
@@ -144,8 +144,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
index 393a6bc..efbf37f 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
@@ -164,8 +164,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFont() {
 	if(SWT.getPlatform() == "win32") {
 		return "Arial";	
-	} else if (SWT.getPlatform() == "motif") {
-		return "Helvetica";		
 	} else if (SWT.getPlatform() == "gtk") {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
index 3e3399c..4a49f41 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
@@ -185,8 +185,6 @@ public void paint(GC gc, int width, int height) {
 static String getPlatformFontFace(int index) {
 	if(SWT.getPlatform() == "win32" || SWT.getPlatform() == "wpf") {
 		return new String [] {"Bookman Old Style", "Century Gothic", "Comic Sans MS", "Impact", "Garamond", "Lucida Console", "Monotype Corsiva"} [index];	
-	} else if (SWT.getPlatform() == "motif") {
-		return new String [] {"urw palladio l", "Courier", "qub", "URW Gothic L", "Times", "Lucida", "URW ChanceryL"} [index];
 	} else if (SWT.getPlatform() == "gtk") {
 		return new String [] {"Luxi Mono", "KacstTitleL", "Baekmuk Batang", "Baekmuk Headline", "KacstFarsi", "Baekmuk Gulim", "URW Chancery L"} [index];
 	} else if (SWT.getPlatform() == "carbon") {
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/texteditor/text.txt b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/texteditor/text.txt
index 61429ef..34bd830 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/texteditor/text.txt
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/texteditor/text.txt
@@ -156,13 +156,12 @@ public class StyledText extends Canvas {
 	Caret defaultCaret = null;
 	boolean updateCaretDirection = true;
 
-	final static boolean IS_CARBON, IS_GTK, IS_MOTIF;
+	final static boolean IS_CARBON, IS_GTK;
 	final static boolean DOUBLE_BUFFER;
 	static {
 		String platform = SWT.getPlatform();
 		IS_CARBON = "carbon".equals(platform);
 		IS_GTK = "gtk".equals(platform);
-		IS_MOTIF = "motif".equals(platform);
 		DOUBLE_BUFFER = !IS_CARBON;
 	}
 
@@ -5062,13 +5061,6 @@ void handleKey(Event event) {
 			// produces characters on the mac.
 			ignore = (event.stateMask ^ SWT.COMMAND) == 0 ||
 					(event.stateMask ^ (SWT.COMMAND | SWT.SHIFT)) == 0;
-		} else if (IS_MOTIF) {
-			// Ignore accelerator key combinations (we do not want to 
-			// insert a character in the text in this instance). Do not  
-			// ignore ALT combinations since this key sequence
-			// produces characters on motif.
-			ignore = (event.stateMask ^ SWT.CTRL) == 0 ||
-					(event.stateMask ^ (SWT.CTRL | SWT.SHIFT)) == 0;
 		} else {
 			// Ignore accelerator key combinations (we do not want to 
 			// insert a character in the text in this instance). Don't  
@@ -6796,7 +6788,7 @@ public void setCaretOffset(int offset) {
  * @see org.eclipse.swt.dnd.Clipboard#setContents
  */
 void setClipboardContent(int start, int length, int clipboardType) throws SWTError {
-	if (clipboardType == DND.SELECTION_CLIPBOARD && !(IS_MOTIF || IS_GTK)) return;
+	if (clipboardType == DND.SELECTION_CLIPBOARD && !IS_GTK) return;
 	TextTransfer plainTextTransfer = TextTransfer.getInstance();
 	TextWriter plainTextWriter = new TextWriter(start, length);
 	String plainText = getPlatformDelimitedText(plainTextWriter);
@@ -7023,7 +7015,6 @@ void setHorizontalScrollBar() {
 		Rectangle clientArea = getClientArea();
 		// only set the real values if the scroll bar can be used 
 		// (ie. because the thumb size is less than the scroll maximum)
-		// avoids flashing on Motif, fixes 1G7RE1J and 1G5SE92
 		if (clientArea.width < lineCache.getWidth()) {
 			horizontalBar.setValues(
 				horizontalBar.getSelection(),
@@ -7172,7 +7163,6 @@ void setScrollBars() {
 		
 		// only set the real values if the scroll bar can be used 
 		// (ie. because the thumb size is less than the scroll maximum)
-		// avoids flashing on Motif, fixes 1G7RE1J and 1G5SE92
 		if (clientArea.height < maximum) {
 			verticalBar.setValues(
 				verticalBar.getSelection(),
diff --git a/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java b/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
index 1426ad8..3499c31 100644
--- a/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
+++ b/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
@@ -57,7 +57,7 @@ public class SnippetLauncher {
 							platform = "JavaXPCOM";
 							skip = true;
 						} else {
-							String [] platforms = {"win32", "motif", "gtk", "photon", "carbon"};
+							String [] platforms = {"win32", "gtk", "photon", "carbon"};
 							for (int p = 0; p < platforms.length; p++) {
 								if (!platforms[p].equals(platform) && source.indexOf("." + platforms[p]) != -1) {
 									platform = platforms[p];
